### ISSUE TITLE
Add Spanish translation for messages

### ISSUE THREAD

**Issue by bartoszlenar** [bartoszlenar/Validot#11]
## Feature description

* Spanish translation for the default Validot's message set.
* Placeholders support where applicable.
* Fluent api extension method to `ValidatorSettings` so it can be nicely used from `Validator.Factory.Create`.

## Feature in action

``` csharp
var validator = Validator.Factory.Create(
    specification,
    s => s.WithSpanishTranslation()
);

validator.Settings.Translations["Spanish"] // this is the full translation dictionary

validator.Validate(model).ToString(translationName: "Spanish") // gets result error output in Spanish
```

## Feature implementation walkthrough

* In Validot, even English language is implemented as a translation that maps the property names from [MessageKey](https://github.com/bartoszlenar/Validot/blob/main/src/Validot/Translations/MessageKey.cs) class to the human-readable phrases.
* Take a look at the files in [src/Validot/Translations](https://github.com/bartoszlenar/Validot/tree/main/src/Validot/Translations) directory. This is the starting point, where the translation should be placed. Create `Spanish` directory inside and then `SpanishTranslation.cs`.
* The content of `SpanishTranslation.cs` could be (and should be...) entirely based on the file [EnglishTranslation.cs](
https://github.com/bartoszlenar/Validot/blob/main/src/Validot/Translations/English/EnglishTranslation.cs). Please copy-paste the content and replace English phrases with the Spanish ones.
   * Please be aware that the particular phrase could contain a placeholder. If you see the English text like `"Must be equal to {value}"`, don't translate the text between curly brackets `{` and `}`. The Polish version for this is `"Musi być równe {value}"`. As you can see, `{value}` stays as it was.
   * As a matter of fact, it would be nice if you add formatting to your phrases. Placeholders will be filled with the values coming from the rule arguments and you can adjust the culture like this: `"Musi być równe {value|culture=pl-PL}"`. So after the variable name (`value`) there is a pipeline character `|` and then parameter (`culture`) with the value `pl-PL` - culture code, the string that goes to the [CultureInfo.GetCultureInfo](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.getcultureinfo?view=netstandard-2.0#System_Globalization_CultureInfo_GetCultureInfo_System_String_) method. To truly understand how it works, please read the [section about message arguments in the docs](https://github.com/bartoszlenar/Validot/blob/main/docs/DOCUMENTATION.md#message-arguments)
* Similarly, next to `SpanishTranslation.cs`, please create `SpanishTranslationExtensions.cs` and copy-paste the content from the `EnglishTranslationExtensions.cs`. Just find-and-replace `English` with `Spanish` should work just fine.
* Creating unit tests are as simple as creating extensions from the point above. Enter [TranslationTests.cs](https://github.com/bartoszlenar/Validot/blob/main/tests/Validot.Tests.Unit/Translations/TranslationTests.cs) file and look for nested class `English` - copy-paste it and replace `English` with `Spanish`. You can notice that `Polish` is done in a similar way.
* Done! Thank you for your contribution!

**Comment by restebanlm** [comment]
Hi @bartoszlenar . I'm going to work on the adaptation to Spanish.

**Comment by bartoszlenar** [comment]
Thank you, @restebanlm. If anything in the issue description is not clear enough, or if you need any additional help or technical guidance, let me know in the comments.

**Comment by restebanlm** [comment]
I hope the pull I sent you is useful.

**Comment by bartoszlenar** [comment]
It's great. Thank you.

Already merged in #18
